1_turb_abl_fsi/template_files/nrel5mw_amr.inp

```diff
@@ -0,0 +1,199 @@
+#.......................................#
+time.stop_time               = -1     # Max (simulated) time to evolve
+time.max_step                = 31440 #-1          # Max number of time steps
+#.......................................#
+#         TIME STEP COMPUTATION         #
+#.......................................#
+time.fixed_dt         =   0.003443526170799   # Use this constant dt if > 0
+time.cfl              =   0.95       # CFL factor
+#.......................................#
+#            INPUT AND OUTPUT           #
+#.......................................#
+io.restart_file = {AMR_RESTART_FILE}
+time.plot_interval            =  500       # Steps between plot files
+time.checkpoint_interval      =  500       # Steps between checkpoint files
+time.plot_start               =  30000
+#.......................................#
+#               PHYSICS                 #
+#.......................................#
+incflo.gravity        =  0.0  0.0 -9.81  # Gravitational force (3D)
+incflo.density        =  1.0             # Reference density
+incflo.use_godunov = 1
+incflo.godunov_type = "weno_z"
+incflo.velocity = 9.8726896031426 5.7 0.0
+
+transport.viscosity = 0.0
+transport.laminar_prandtl = 0.7
+transport.turbulent_prandtl = 0.3333
+
+turbulence.model = OneEqKsgsM84
+TKE.source_terms = KsgsM84Src
+
+incflo.physics = ABL
+ICNS.source_terms = BoussinesqBuoyancy CoriolisForcing BodyForce ABLMeanBoussinesq
+
+BoussinesqBuoyancy.reference_temperature = 300.0
+BoussinesqBuoyancy.read_temperature_profile = true
+BoussinesqBuoyancy.tprofile_filename = {AMR_TPROFILE_FILE}
+
+CoriolisForcing.east_vector = 1.0 0.0 0.0
+CoriolisForcing.north_vector = 0.0 1.0 0.0
+CoriolisForcing.latitude = 40.0
+CoriolisForcing.turn_off_vertical_force = True
+CoriolisForcing.rotational_time_period = 86400.0
+
+BodyForce.magnitude = -0.000349710865169856 0.0011134941091590048 0.0
+
+ABL.wall_shear_stress_type = "local"
+ABL.inflow_outflow_mode = true
+ABL.wf_velocity = 6.093275678667113 3.659473797517521
+ABL.wf_vmag = 7.12794309269604
+ABL.wf_theta = 300.2026467204779
+
+ABL.reference_temperature = 300.0
+ABL.temperature_heights = 0.0 750.0 850.0 2000.0
+ABL.temperature_values = 300.0 300.0 308.0 311.45
+ABL.perturb_temperature = true
+ABL.cutoff_height = 50.0
+ABL.perturb_velocity = true
+ABL.perturb_ref_height = 50.0
+ABL.Uperiods = 4.0
+ABL.Vperiods = 4.0
+ABL.deltaU = 1.0
+ABL.deltaV = 1.0
+ABL.kappa = .41
+ABL.surface_roughness_z0 = 0.01
+ABL.surface_temp_flux = 0.005
+ABL.normal_direction = 2
+ABL.log_law_height = 5.0
+ABL.stats_output_frequency = 1
+ABL.stats_output_format = netcdf
+
+ABL.enable_hybrid_rl_mode = true
+#.......................................#
+#        ADAPTIVE MESH REFINEMENT       #
+#.......................................#
+amr.n_cell              = 512 512 192    # Grid cells at coarsest AMRlevel
+amr.max_level           = 4            # Max AMR level in hierarchy
+amr.max_grid_size       = 128
```

Comment:
Does this change with the machine?

Reply:
I'm not sure.  We can update it to a scripted parameter if need be.